mls-rs/src/group/proposal_filter/filtering.rs

```diff
@@ -169,10 +175,15 @@ where
         group_extensions_in_use: &ExtensionList,
         commit_time: Option<MlsTime>,
     ) -> Result<ProposalBundle, MlsError> {
+        let member_validation_context = MemberValidationContext::ForCommit {
+            current_context: self.original_context,
+            new_extensions: group_extensions_in_use,
```

Comment:
can we change this variable name, feels like it would be the original extensions, but it isn't